src/library_webgl.js

```diff
@@ -739,6 +766,65 @@ for (/**@suppress{duplicate}*/var i = 0; i < {{{ GL_POOL_TEMP_BUFFERS_SIZE }}};
       disableHalfFloatExtensionIfBroken(ctx);
 #endif
 
+      // If end user enables *glGetProcAddress() functionality, then we must filter out
+      // all future WebGL extensions from being passed to the user, and only restrict to advertising
+      // extensions that the *glGetProcAddress() function knows to handle.
+#if GL_ENABLE_GET_PROC_ADDRESS
+      ctx._allSupportedExtensions = ctx.getSupportedExtensions;
```

Comment:
Rather than interposing like this, can we instead explicitly call our wrapper 
function where it makes sense?   (then we can avoid this caching of the original function).

Reply:
I think this is cleaner this way, as there will be just one code location where this is maintained. There are several places otherwise where this needs to be patched in, which would create dependencies across library_glemu.js, library_html5_webgl.js, library_webgl.js, webGLClient.js and webGLWorker.js.